distribution/docs/src/main/resources/content/_managing/_configuring/hardening-solr.adoc

```diff
@@ -80,38 +79,6 @@ is not necessary for cleartext. Cleartext is used by the system exactly as it ap
 <<_do_not_autogenerate_a_solr_password, follow these instructions>>.
 . Finally, restart ${branding}
 
-==== Restore the Default Password in Solr
```

Comment:
:exclamation:  Are password recovery instructions no longer needed?

Reply:
The password is now set when they follow the Solr documentation for installing and securing SolrCloud.

Reply:
1. It sounds like we no longer want DDF trying to change the Solr password. If that is true, we should remove the module: 
`ddf/platform/solr/solr-password` 

2. And remove the interface `ddf/platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/admin/core/api/jmx/SystemPropertiesAdminInterceptor.java`,

3. as well as the reference to it in `SystemPropertiesAdmin`

4. Which means we can also remove the property `solr.attemptAutoPasswordChange=true` from the system properties file.

Reply:
Wait, I can remove more code?  :100:

Reply:
Yes. The code I spent weeks writing and re-writing.